0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -96,52 +174,102 @@ Content-Type: application/json
+  "invoice_info": "https://www.amazon.com/gp/your-account/order-details?ie=UTF8&orderID=111-7777777-1111111"
 }
 ```
 
-### Quoting
+| Field | Type | Description |
+|---|---|---|
+| `type` | `"invoice"` | Receiver type |
+| `account` | ILP Address | ILP Address of the recipient's account |
+| `currency_code` | [ISO 4217](https://en.wikipedia.org/wiki/ISO_4217) code | Currency code to identify the invoice's currency |
+| `currency_symbol` | String | Symbol for the invoice's currency intended for display in the sender's UI (e.g. `"$"` or `"shares"`) |
+| `amount` | Decimal String | Value of the invoice in the recipient's currency |
+| `status` | Enum: `"paid"`, `"unpaid"`, `"cancelled"` | State of the invoice |
+| `invoice_info` | URI | URI where additional information about the invoice can be found. |
 
-The sender requests quotes from neighboring connectors using [ILQP](../0008-interledger-quoting-protocol/).
+### Setup (`POST <receiver>`)
 
-### Setup
+When the sender is ready to make a payment, it submits a payment object to the receiver.
 
-When the sender is ready to make a payment, it submits a payment object to the receiver:
+#### Request
+
+##### For a Payee
 
 ``` http
-POST /api/receiver/bob/payments HTTP/1.1
+POST /api/receiver/bob HTTP/1.1
 Host: red.ilpdemo.org
 Accept: application/json
 
 {
   "amount": "10.40",
-  "source_identifier": "alice@blue.ilpdemo.org",
+  "sender_identifier": "alice@blue.ilpdemo.org",
   "memo": "Hey Bob!"
 }
 ```
+
+| Field | Type | Description |
+|---|---|---|
+| `amount` | Decimal String | _Destination amount_ the receiver will receive |
+| `sender_identifier` | String | Identifier of the sender |
+| `memo` | String | Message for the recipient linked to the payment |
+
+##### For an Invoice
+
+``` http
+POST /api/receiver/bob HTTP/1.1
+Host: red.ilpdemo.org
+Accept: application/json
+
+{
+  "sender_identifier": "alice@blue.ilpdemo.org"
+}
+```
+
+| Field | Type | Description |
+|---|---|---|
+| `sender_identifier` | String | Identifier of the sender |
+
+#### Response
 ``` http
 HTTP/1.1 201 Created
 Content-Type: application/json
 
 {
-  "receipt_condition": "cc:1:1:47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU:1"
+  "packet": {
+    "account": "ilpdemo.red.bob",
+    "amount": "10.40",
+    "data": {
+      "sender_identifier": "alice@blue.ilpdemo.org",
+      "memo": "Hey Bob!"
+    }
+  },
+  "condition": "cc:0:3:47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU:0",
+  "additional_routing_info": {
+    "something": "that helps the sender's client deliver the packet to the recipient"
+  }
 }
 ```
+| Field | Type | Description |
+|---|---|---|
+| `packet` | Object | JSON representation of the ILP Packet used by the sender's client to construct the binary ILP Packet |
+| `condition` | Crypto Condition | Execution condition the sender must use for the payment to the recipient |
+| `additional_routing_info` | Object | Additional info used by the sender's client to route the payment to the recipient |
```

Comment:
We already know we're going to need the receiver to specify the rate curve from a well known ledger to their ledger, but I'm not entirely sure about the data format. It's probably a good idea to specify it more precisely here, since clients would need to understand that.